packages/react/src/generators/library/library.ts

```diff
@@ -103,7 +107,7 @@ export async function libraryGeneratorInternal(host: Tree, schema: Schema) {
   tasks.push(lintTask);
 
   // Set up build target
-  if (options.buildable && options.bundler === 'vite') {
```

Comment:
I see that you marked buildable as deprecated, and at the same time have removed these references to it.

Is bundler already always being set reliably? Is removing buildable a safe thing to do? (e.g. it's deprecated but not removed)

Reply:
bundler has a default value none. if the bundler is none, it will not add a build target regardless buildable value.

Reply:
changed it back